MAUI/StepProgressBar/description.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: post
+title: Description in .NET MAUI Step ProgressBar control | Syncfusion
+description: Learn here all about Description support in Syncfusion .NET MAUI Step ProgressBar (SfStepProgressBar) control and more.
+platform: maui
+control: SfStepProgressBar
+documentation: ug
+---
+
+# Description in .NET MAUI StepProgressBar (Step Progress Bar)
+Each step in a multi-step process has a different operation. To display information about a step's status can be shown on either side. A primary description will be on the right or bottom of the step, and a secondary description will be on the left or top of the step.
+
+## StepProgressBar overview
+The following overview image illustrates the major elements presented in StepProgressBar.
```

Comment:
Needed?

Reply:
Removed the overview line, because we have specify the text in separate topic